Contents/Code/__init__.py

```diff
@@ -74,15 +74,40 @@ def Search(self, results, media, lang, manual, movie):
 
         # http://127.0.0.1:8111/api/serie/search?query=Clannad&level=1&apikey=d422dfd2-bdc3-4219-b3bb-08b85aa65579
 
-        prelimresults = HttpReq("api/serie/search?query=%s&level=%d&fuzzy=%d" % (urllib.quote(name.encode('utf8')), 1, Prefs['Fuzzy']))
+        if movie:
+            if media.filename:
+                filename = media.filename.split('%2F')[-1]
```

Comment:
That would work, yes. This wouldn't
```
>>> import os
>>> os.path.basename('/path%2Fto%2Ffilename.mkv')
'path%2Fto%2Ffilename.mkv'
```
Blame plex for storing it as such. It stores `%2F` in `media.filename` instead of `/`

Reply:
Wait. What about Windows then? Damn it.. I need to go check.

Reply:
I fixed it. I probably should've looked up before what `%2F` was..